### ISSUE TITLE
Error logging in: Steam returned "GeneralFailure" 2023

### ISSUE THREAD

**Issue by torrenars** [Jessecar96/SteamDesktopAuthenticator#798]
I have 4 account that has knives and skins, I cannot use confirmation because I need to login again and refresh session but both not working I get this error instead, I badly need to sell the knives and skins I have please help me how to fix this problem https://prnt.sc/VCImKxBRVz8M

**Comment by dyc3** [comment]
A GeneralFailure usually occurs because Valve's services are very unreliable. Try waiting a while and try again.

**Comment by SadSisyphus** [comment]
I have exactly the same problem for two days now and it has not been solved by waiting, as suggested by dyc3. Are you sure it should just wait or should I do something specific? 

I uninstalled and installed the app again and it still doesn't work. I don't know what could be happening, the only thing that I think is weird is that I have a very long password, more than 60 characters, this can cause some error?

Translated with www.DeepL.com/Translator (free version)

**Comment by SadSisyphus** [comment]
I have exactly the same problem for two days now and it has not been solved by waiting, as suggested by dyc3. Are you sure it should just wait or should I do something specific?

I uninstalled and installed the app again and it still doesn't work. I don't know what could be happening, the only thing that I think is weird is that I have a very long password, more than 60 characters, this can cause some error?

**Comment by dyc3** [comment]
Yes, Steam services are just that unreliable from my experience.

**Comment by sneaky-0** [comment]
i am facing the issue that when i try relogin it pops a captcha and i have tried solving it multiple times but it just sayd general failiure

**Comment by Snesik** [comment]
Similarly, captcha crashes 2 times, then general failure

**Comment by RATT78** [comment]
Same is happening to me since today, i try to put some trading card to sell on the community market and get the error that asking me to refresh my session (it happen to me so many times) but this time the LOGIN AGAIN did not works, i get some captchas and after that get the "GeneralFailure" error message.

**Comment by sneaky-0** [comment]
i am not able to add new accounts too it ask for phone sms again and again

**Comment by RATT78** [comment]
I'll try to be patient and wait until next Tuesday when steam does the weekly maintenance to see if it's requested, as I said before, I've had to do relogin/refresh countless times since I've used SDA, but it always works, and I've never have to solved any captchas (1st time i see this on SDA). To tell the truth, I'm worried because this happens to me in my main account (the one i use to sell items on the community market) while I haven't had any problem in the ALT accounts, yesterday I was able to trade several cards from Capcom Stadium 2 that was FREE from those ALT accounts (which also use SDA) to my main account.

**Comment by RATT78** [comment]
Same, this issue only affect one of my accounts, no fix by now, keep getting the same error, im using SDA since 1.0.2 and this is the first time a "Login Again" did not works.

**Comment by RATT78** [comment]
HEY! It fixes "alone", well i was using the 1.0.8 but now i try to relogin using the 1.0.10 and it works prefectly.

**Comment by lnickersl** [comment]
Hi there, getting "General Failure" error on "Login Again" as well, have this issue for 3 days now, only 1 account has this issue right now.

Tried downloading another 1.0.10 in different folder, copied "maFiles", same result.

Switched to 1.0.10 more than a year ago, "Login again" worked before.

**Comment by lolifuta** [comment]
you probably need to use the same ip as your steam region, it useful for me

**Comment by lnickersl** [comment]
> you probably need to use the same ip as your steam region, it useful for me

It's the same for me.

**Comment by Jianchitz** [comment]
> > you probably need to use the same ip as your steam region, it useful for me
> 
> It's the same for me.

hey did you find a solution?

**Comment by Jianchitz** [comment]
> Hello it fixed "alone" for me, just in case i will told you what i did before SDA relogin works in my case, today i start by luanching the old version i keep using since the auto update stop working for me, the 1.0.8 version, i try a relogin and as always get the general failure from steam error, i got to the community store and buy a trading card that i need to complete a set, i really was no hope but give a try to the 1.0.10 (i try with this previously with no results) and this time no captchas, no error, just the window asking me to write the encryption key and .... TADA!!!! my session was succesfully login and now im able to acept/send trades and place items to sell in the market.

Can you explain how did you fix it again?

**Comment by Jianchitz** [comment]
> A GeneralFailure usually occurs because Valve's services are very unreliable. Try waiting a while and try again.

hey ppl been complaining about this for a month now is it really bcs of steam? can any admin check it please :(

**Comment by HALLMA2K** [comment]
New user also having this issue. Ill keep an eye out here for a solution.

**Comment by Miklos1999** [comment]
hello i tryed every possible thing to do, to fix this. and found A solution!
step1. close all tabs
step2. turn on vpn to a country close to your location
step3. open csgo/any steam game while running the vpn
step4. open Desktop Authenticator and login again.
you should then get a succesfull login. then close game and Authenticator. turn off vpn and you are good to go again.

**Comment by lnickersl** [comment]
My issue went away somehow, I don't think I changed anything.

**Comment by NafixXx** [comment]
> hello i tryed every possible thing to do, to fix this. and found A solution! step1. close all tabs step2. turn on vpn to a country close to your location step3. open csgo/any steam game while running the vpn step4. open Desktop Authenticator and login again. you should then get a succesfull login. then close game and Authenticator. turn off vpn and you are good to go again.

Omg dude i FUCKING LOVE U. You my best friend now. Мb my Eng isn't good, but i'll try. Thanks so much, i very appreciative you. You very helped me a lot. I can't  made trade offers for 4 weeks.  When i became billioner, i'll give you 1m$ i promise. I never forget you.

**Comment by Miklos1999** [comment]
> > hello i tryed every possible thing to do, to fix this. and found A solution! step1. close all tabs step2. turn on vpn to a country close to your location step3. open csgo/any steam game while running the vpn step4. open Desktop Authenticator and login again. you should then get a succesfull login. then close game and Authenticator. turn off vpn and you are good to go again.
> 
> Omg dude i FUCKING LOVE U. You my best friend now. Мb my Eng isn't good, but i'll try. Thanks so much, i very appreciative you. You very helped me a lot. I can't made trade offers for 4 weeks. When i became billioner, i'll give you 1m$ i promise. I never forget you.

looking forward for that milli :D
